cmd/kops/rollingupdatecluster.go

```diff
@@ -51,7 +52,22 @@ var (
 
 	Use export KOPS_FEATURE_FLAGS="+DrainAndValidateRollingUpdate" to use beta code that drains the nodes
 	and validates the cluster.  New flags for Drain and Validation operations will be shown when
-	the environment variable is set.`))
+	the environment variable is set.
+
+	Node Replacement Strategies Alpha Feature
+
+	We are now including three new strategies that influence node replacement. All masters and bastions
+	are rolled sequentially before the nodes, and this flag does not influence their replacement.  These
+	strategies utilize the feature flag mentioned above.
+
+	1. "default" - A node is drained then deleted.  The cloud then replaces the node automatically.
+	2. "create-all-new-ig-first" - All node instance groups are duplicated first; then all old nodes are cordoned.
```

Comment:
Do we have `create-all-new-ig-first` before `create-new-by-ig` only cause it's faster?